src/js/game/LevelMVC/LevelView.js

```diff
@@ -769,6 +782,18 @@ module.exports = class LevelView {
     }
   }
 
+  /**
+  * Handling the first case of walking onto a track while not currently on one
+  */
+  firstTime(currPos, nextPos) {
```

Comment:
nit: this method should be much more specifically-named; something like `isFirstTimeOnRails`